modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for some connections within {product-title-short} and between {product-title-short} and external components are shown in the following table. You might need this information for configuring the firewalls in your system to allow traffic on certain ports.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by sensor to Central's port 443.
+
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS or gRPC
+| 443
+|
+
+| Central and user (CLI)
+| gRPC
+| 443
+|
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+|
+
+| Scanner and Scanner-DB
+| TCP
+| 5432
+|
+
+| Sensor and admission controller
+| gRPC
+| 443
+| This is a bidirectional gRPC stream, initiated by the admission controller to sensor's port 443.
+
+
+| Sensor and collector
+| gRPC
+| 443
+| This connection is bidirectional and initiated by collector to Central's port 443. If node scanning is enabled for {ocp}, a connection on port 8444 is initiated by sensor to compliance (running in the collector pod).
```

Comment:
```suggestion
| This connection is bidirectional and initiated by collector to Sensor's port 443. If node scanning is enabled for {ocp}, a connection on port 8444 is initiated by Sensor to compliance (running in the collector pod).
```

Reply:
Regarding bidirectionally: it will be bidirectional in 4.6, now it is mainly Collector -> Sensor.
Sensor can however communicate with Compliance container (living inside the Collector pod), so I guess this is bidirectional in 4.5 and earlier.